samples/interactive-tutorials/src/test/java/events/setup/EventsCreateBigQueryTableTest.java

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
Probably should not add tests for setup / cleanup like before, since this is a script that runs only once to set up the test env on the DEP project. If we run the tests with other tests concurrently, it may be flaky. e.g. the clean up tests may delete all the products that causing other product import tests to fail.

Reply:
Removed setup/cleanup tests.